### ISSUE TITLE
[BUG] Voxel Lighting in one game when future lighting is enabled.

### ISSUE THREAD

**Issue by JustDreaded** [pizzaboxer/bloxstrap#3076]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.
- [X] I have thoroughly looked through the available Wiki articles and could not find a solution to my problem.
- [X] I am using the latest version of Bloxstrap.
- [ ] I did not answer truthfully to all the above checkboxes.

### What problem did you encounter?

I have future lighting 3 enabled but this one game, "After the Flash: Wintertide" has Voxel Lighting. They're saying Roblox forced this but it's not working. I tried switching the lighting mode from another mode but that didn't work at all. Is there a way to fix this?

**Comment by pizzaboxer** [comment]
How are you determining that the lighting isn't changing?

**Comment by pizzaboxer** [comment]
> Voxel lighting is extremely bright on avatars and future lighting isn't.

That's not an accurate way of determining it. Press Shift+F2 in a game and show a screenshot of the details that appear when ingame.

**Comment by JustDreaded** [comment]
It says it's shadow map even though it's Voxel. 
<img width="210" alt="Screenshot 2024-10-07 200009" src="https://github.com/user-attachments/assets/84d7f91a-5f3b-4a36-ac1a-9a32cf67befd">

**Comment by user1126s** [comment]
bruh now you just insisting

**Comment by pizzaboxer** [comment]
> It says it's shadow map even though it's Voxel.

No, it's ShadowMap. Just to be clear, do you still have your lighting set to Future?

**Comment by JustDreaded** [comment]
Yes, I do have it set to future.

**Comment by pizzaboxer** [comment]
Upload your configuration please: https://github.com/pizzaboxer/bloxstrap/wiki/Uploading-your-Bloxstrap-configuration
